### PR TITLE
Added while loop to aut generate divs

### PR DESCRIPTION
**Pull request by waseembrare**
- created a php while loop to autogenerate divs for item cards. 
- added a html element

### CODE REVIEW

**Review by damienscarpellini — APPROVED** [review]
looks good - approved. on a side note, not sure if its unconventional, but you haven't put any semicolons within your php tags thats mixed in with your HTML.

File: index.php [hunk 1]
```diff
@@ -1,29 +1,18 @@
...
 
 <!DOCTYPE html>
 <html lang="en-GB">
 <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>cheffin' it up bruv</title>
     <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=JetBrains+Mono&display=swap" rel="stylesheet">
+    <link href="https://fo2nts.googleapis.com/css2?family=JetBrains+Mono&display=swap" rel="stylesheet">
```
Comment (mtmgoliath) [inline index.php:15]: is this right?

Comment (waseembrare) [reply index.php:15]: no lol i'll change now
- great spot

**Review by mtmgoliath — APPROVED** [review]

File: index.php (hunk 1 above)
Comment (mtmgoliath) [reply index.php:15]: np, looks good otherwise =]

Comment (waseembrare) [reply index.php]: i've pushed new file

File: test/functions.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<?php
+
+require '../functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class Index extends TestCase
+{
+
+    public function testDisplayItemCards()
```
Comment (mtmgoliath) [inline test/functions.php:10]: testSuccessDisplayItemCards()

**Pull request merged**